engine/src/main/java/org/opencds/cqf/cql/engine/elm/execution/ValueSetRefEvaluator.java

```diff
@@ -16,10 +15,10 @@
     public static ValueSet toValueSet(Context context, ValueSetRef vsr) {
         boolean enteredLibrary = context.enterLibrary(vsr.getLibraryName());
         try {
-            ValueSetDef vsd = context.resolveValueSetRef(vsr.getLibraryName(), vsr.getName());
+            ValueSetDef vsd = context.resolveValueSetRef(vsr.getName());
             ValueSet vs = new ValueSet().withId(vsd.getId()).withVersion(vsd.getVersion());
             for (CodeSystemRef csr : vsd.getCodeSystem()) {
-                CodeSystemDef csd = context.resolveCodeSystemRef(csr.getLibraryName(), csr.getName());
+                CodeSystemDef csd = context.resolveCodeSystemRef(csr.getName());
```

Comment:
This needs to enter the library context for the csr.getLibraryName(), the code system reference might be to a different library than the valuesetref is declared in.